bin/autoproj_bootstrap

```diff
@@ -601,4 +601,4 @@ if !ops.skip_stage2?
     ops.call_stage2
 end
 ops.run_autoproj 'bootstrap', *bootstrap_options
```

Comment:
> Not sure it's really useful to rescue in this case, since you will get your non-zero exit status.

... and a not very descriptive uncaught exception error message. Is that desirable in a script?